megazords/full/src/lib.rs

```diff
@@ -25,6 +25,7 @@ pub use suggest;
 pub use sync_manager;
 pub use tabs;
 pub use viaduct;
+pub use webext_storage;
```

Comment:
We probably need to work out a way to arrange for the local kotlin tests to work without adding it to the megazord at this point as it will end up shipped on Android even while unused. I'll think about this but it should be doable. It's fine to keep it for now, just not for actually landing it.